app/experimenter/experiments/constants/nimbus.py

```diff
@@ -505,6 +505,62 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_REGULAR_USER_NEED_PIN = NimbusTargetingConfig(
+    name="Regular user (need pin)",
+    slug="regular_user_need_pin",
+    description="Between 14 and 20 days days of activity in the past 28 days needing pin",
+    targeting="{regular_user} && doesAppNeedPin || {sticky}".format(
+        regular_user=TARGETING_REGULAR_USER_URIS.targeting,
+        sticky=TARGETING_STICKY,
+    ),
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
```

Comment:
nit: put each of these next to where other related activity level targeting are (so infrequent then casual then regular then core)